expat/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: expat
+  version: 2.1.0
+
+source:
+  fn: expat-2.1.0.tar.gz
+  url: http://sourceforge.net/projects/expat/files/expat/2.1.0/expat-2.1.0.tar.gz
+  md5: dd7dab7a5fea97d2a6a43f511449b7cd
+
+requirements:
+  build:
+    - gcc
```

Comment:
@ccordoba12 I don't get the logic at all. System-wide packages are less reliable than conda dependency, but you suggest to choose between (1) loosing `gcc` dependency or (2) adding the unnecessary `libgcc` dependency for the sake of "custom". Python is built with gcc isn't it? Still, it doesn't require `libgcc`.